src/frontends/pytorch/src/utils.hpp

```diff
@@ -52,7 +52,8 @@ op::PadType convert_pad(const std::string& pt_pad);
 
 Output<Node> concat_list_construct(const Output<Node>& input);
 
-OutputVector make_framework_node(const NodeContext& context);
+OutputVector make_framework_node_ignore_bodies(const NodeContext& context, const std::string& exception);
+OutputVector make_framework_node(const NodeContext& context, const std::string& exception);
 
```

Comment:
Please provide tests for the report to cover cases with different exceptions, etc.

Reply:
What kind of tests is expected here? Testing exact wording of error I think is useless

Reply:
Not useless, you just validate that all unsupported operations and exceptions are printed. And nothing is missed.
We validate this in TF FE and quite useful test

Reply:
Added test